legion/docs/source/command_utils.md

```diff
@@ -0,0 +1,119 @@
+# Utils
```

Comment:
What is exactly the point of having command line help output from utilities put into documentation?